components/PriceFeedTable.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState, useEffect } from "react";
+import {
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+  TextField,
+  Select,
+  MenuItem,
+  Box,
+  Snackbar,
+} from "@mui/material";
+import { HermesClient } from "@pythnetwork/hermes-client";
+import copy from "copy-to-clipboard";
+
+interface PriceFeed {
+  id: string;
+  name: string;
+  assetType: string;
+}
+
+export function PriceFeedTable() {
+  const [priceFeeds, setPriceFeeds] = useState<PriceFeed[]>([]);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [selectedAssetType, setSelectedAssetType] = useState("All");
+  const [openSnackbar, setOpenSnackbar] = useState(false);
+
+  useEffect(() => {
+    const fetchPriceFeeds = async () => {
+      const hermesClient = new HermesClient("https://hermes.pyth.network");
+      const feeds = await hermesClient.getPriceFeeds();
+      const transformedFeeds = feeds.map((feed) => ({
+        id: feed.id,
+        name: feed.attributes.display_symbol || "",
+        assetType: feed.attributes.asset_type || "",
+      }));
+      setPriceFeeds(transformedFeeds);
+    };
+
+    fetchPriceFeeds();
+  }, []);
+
+  const filteredData = priceFeeds.filter((feed) => {
+    const matchesSearch =
+      feed.id.toLowerCase().includes(searchTerm.toLowerCase()) ||
+      feed.name.toLowerCase().includes(searchTerm.toLowerCase());
+    const matchesAssetType =
+      selectedAssetType === "All" || feed.assetType === selectedAssetType;
+    return matchesSearch && matchesAssetType;
+  });
+
+  const handleCopy = (id: string) => {
+    copy(id);
+    setOpenSnackbar(true);
+  };
+
+  return (
+    <Box>
+      <Box sx={{ mb: 3, display: "flex", gap: 2 }}>
```

Comment:
The `Box` component isn't useful with tailwind; these styles can be directly added to a div by using the correct utility classes, e.g. something like `<div className="mb-1 flex gap-2">`